### PR TITLE
docs: update getting started guide after validation

### PR DESCRIPTION
**Pull request by salasberryfin**
### Descrition

After validating [Getting Started](https://docs.rancher-turtles.com/docs/category/getting-started), we identified a few improvements:
- If disabling `embedded-cluster-api` during Rancher installation, the cluster must be `clusterctl` initialized to successfully install Rancher. If it is not, it will fail. The problem with this approach is that `clusterctl init` will create namespaces that we're then creating again in the `rancher-turtles` Helm chart. Hence, we decided to document a "standard" installation of Rancher (with `embedded-cluster-api` enabled) and then disabling the feature manually. This fixes the problem with the namespaces.
- Installing `rancher-turtles` after Rancher means that `cert-manager` is already installed in the cluster. If no flag is passed to `rancher-turtles` Helm chart, it'll try to install `cert-manager` as a dependency. It is now recommended to disable this.
- The Helm chart is published with the release workflow but the version must be specified in the `helm install` command.

### CODE REVIEW

File: docs/getting-started/rancher.md [hunk 1]
```diff
@@ -4,25 +4,34 @@ sidebar_position: 2
...
+*If you're already running Rancher, you can skip this section and jump to [Setting up Rancher for Rancher Turtles](#setting-up-rancher-for-rancher-turtles).*
 
-1. First, make sure to follow one of the official [installation guides](https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade) for `Rancher`.
-2. When installing `Rancher` using the `helm` command, use the `--set` option to specify the `features` parameter. For the `embedded-cluster-api` feature, set the value to `false` to disable it.
-3. Use the `--version` option to specify the version of `Rancher` you want to install. In this case, use the [recommended](../getting-started/intro.md#prerequisites) `Rancher` version for `Rancher Turtles`.
+Helm is the recommended way to install `Rancher` in an existing or new Kubernetes cluster.
 
-Here's the complete command to install `Rancher` with the `embedded-cluster-api` feature disabled. Replace `<rancher-hostname>` with the actual hostname of your `Rancher` server:
+:::tip
+Make sure to follow one of the official [installation guides](https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade).
```
Comment (furkatgofurov7) [inline docs/getting-started/rancher.md:14]:
```suggestion
Make sure to follow one of the official [installation guides](https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade) for `Rancher`.
```

File: docs/getting-started/rancher.md [hunk 2]
```diff
@@ -4,25 +4,34 @@ sidebar_position: 2
...
+helm install rancher rancher-stable/rancher
+    --namespace cattle-system
+    --create-namespace
+    --set hostname=<rancher-hostname>
+    --version <rancher-version>
+    --wait
 ```
 
-## Existing Rancher installation
+Replace `<rancher-hostname>` with the actual hostname of your `Rancher` server and use the `--version` option to specify the version of `Rancher` you want to install. In this case, use the [recommended](../getting-started/intro.md#prerequisites) `Rancher` version for `Rancher Turtles`:
```
Comment (furkatgofurov7) [inline docs/getting-started/rancher.md:28]:
```suggestion
Replace `<rancher-hostname>` with the actual hostname of your `Rancher` server and use the `--version` option to specify the version of `Rancher` you want to install. In this case, use the [recommended](../getting-started/intro.md#prerequisites) `Rancher` version for `Rancher Turtles`
```

File: docs/getting-started/rancher.md [hunk 3]
```diff
@@ -4,25 +4,34 @@ sidebar_position: 2
 
 # Rancher Setup
 
-## Installing Rancher
+### Installing Rancher
```
Comment (furkatgofurov7) [inline docs/getting-started/rancher.md:7]: Q: why is this needed (jumping from h1 to h3)?

**Review by furkatgofurov7 — COMMENTED** [review]
Overall LGTM, few suggestions and 1 question inline:

File: docs/getting-started/rancher.md (hunk 3 above)
Comment (salasberryfin) [reply docs/getting-started/rancher.md]: This was an error. Thanks for noticing! 🙂

**Comment by salasberryfin** [comment]
Updated with suggestions and ready to be reviewed!

File: docs/getting-started/install_turtles_operator.md [hunk 4]
```diff
@@ -18,13 +18,19 @@ helm repo update
...
     -n rancher-turtles-system
     --dependency-update
     --create-namespace --wait
     --timeout 180s
 ```
 
+:::note
+- If `cert-manager` is already available in the cluster, you can disable its installation as a Rancher Turtles dependency to avoid conflicts:
+`--set cluster-api-operator.cert-manager.enabled=false`
+- For a list of Rancher Turtles versions, refer to [Releases](https://github.com/rancher-sandbox/rancher-turtles/releases).
```
Comment (furkatgofurov7) [inline docs/getting-started/install_turtles_operator.md:31]: nit:
```suggestion
- For a list of Rancher Turtles versions, refer to [Releases page](https://github.com/rancher-sandbox/rancher-turtles/releases).
```

**Review by furkatgofurov7 — APPROVED** [review]

**Review by richardcase — APPROVED** [review]
This looks good, thanks @salasberryfin

**Comment by Danil-Grigorev** [comment]
I believe this issue appeared due to https://github.com/rancher-sandbox/rancher-turtles/pull/93/files removing CAPI Operator helm setup and replacing it with dependency install.

**Comment by salasberryfin** [comment]
> I believe this issue appeared due to https://github.com/rancher-sandbox/rancher-turtles/pull/93/files removing CAPI Operator helm setup and replacing it with dependency install.

@Danil-Grigorev yes, these namespaces are required when `cluster-api-operator` is enabled.

**Review by Danil-Grigorev — APPROVED** [review]

File: docs/getting-started/rancher.md (hunk 3 above)
Comment (Danil-Grigorev) [reply docs/getting-started/rancher.md]: That’s why a visual inspection on the PR is needed. https://github.com/rancher-sandbox/rancher-turtles-docs/pull/32 Minor change of hn may have a very profound effect on docs visual style, difficult to pickup on the diff alone.
